pillar/params.sls

```diff
@@ -37,7 +37,7 @@ kube_group:       'kube'
 kube_log_level:   '2'
 
 # install the addons (ie, DNS)
-addons:           'false'
+addons:           'true'
```

Comment:
Rather than having a global addons=true, it would be nice to see this changed to:

    addons:
      dns: true

so we can add others if necessary easily :)